ADC_Joystick_RTApp_MT3620_BareMetal/.vscode/settings.json

```diff
@@ -1,12 +1,12 @@
-{
-    "cmake.generator": "Ninja",
-    "cmake.buildDirectory": "${workspaceRoot}/out/${buildType}-${command:azuresphere.AzureSphereTargetApiSet}",
-    "cmake.buildToolArgs": [ "-v" ],
-    "cmake.configureSettings": {
-        "CMAKE_TOOLCHAIN_FILE": "${command:azuresphere.AzureSphereSdkDir}/CMakeFiles/AzureSphereRTCoreToolchain.cmake",
-        "ARM_GNU_PATH": "${command:azuresphere.ArmGnuPath}",
-        "AZURE_SPHERE_TARGET_API_SET": "latest-beta"
-    },
-    "cmake.configureOnOpen": true,
-    "C_Cpp.default.configurationProvider": "vector-of-bool.cmake-tools"
+{
+    "cmake.generator": "Ninja",
+    "cmake.buildDirectory": "${workspaceRoot}/out/${buildType}",
+    "cmake.buildToolArgs": [ "-v" ],
+    "cmake.configureSettings": {
+        "CMAKE_TOOLCHAIN_FILE": "${command:azuresphere.AzureSphereSdkDir}/CMakeFiles/AzureSphereRTCoreToolchain.cmake",
+        "ARM_GNU_PATH": "${command:azuresphere.ArmGnuPath}",
+        "AZURE_SPHERE_TARGET_API_SET": "latest-lts"
```

Comment:
I think you can remove this line altogether?  The MS HelloWorld RT sample doesn't have this.

Reply:
I get the following error if I remove this line:
```
Error: Required azure sphere settings not found. Please configure in settings.json or run command 'Azure Sphere: Configure settings' to apply default settings.
```